docs/installation_guide.rst

```diff
@@ -354,7 +354,7 @@ Get `sqlcipher <https://www.zetetic.net/sqlcipher/>`_ version 4:
 
 - If you are running Archlinux you can install the `package <https://www.archlinux.org/packages/community/x86_64/sqlcipher/>`_ with ``pacman``.
 
-- If you are running Ubuntu, at the time of writing of this article Ubuntu is still using sqlcipher v3 which is not supported by rotki. So you should build sqlcipher v4 by hand. We have a script for that which is also used by our CI. Check it out `here <https://github.com/rotki/rotki/blob/7573bcbd9bfc83e0ef352701ae2772a040b4ab5b/install_deps.sh>`__. Install libssl-dev and tclsh by running ``sudo apt-get install libssl-dev tclsh`` if not already installed.
+- If you are running Ubuntu, at the time of writing of this article Ubuntu is still using sqlcipher v3 which is not supported by rotki. So you should build sqlcipher v4 by hand. We have a script for that which is also used by our CI. Check it out `here <https://github.com/rotki/rotki/blob/develop/install_deps.sh>`__. If you already have an older version of sqlcipher run this script with --upgrade flag. Install libssl-dev and tclsh by running ``sudo apt-get install libssl-dev tclsh`` if not already installed.
```

Comment:
Don't point to develop in the docs. Point to the hash of the commit since there may be a reorganization at any point in time in the future of develop and then the link will break.

Reply:
I've changed links so that it points to my commit and right now github shows warning because this commit doesn't belong to the rotki/rotki repository but I hope that when PR will be accepted this warning won't be showing up. Is it okay?

Reply:
yep as long as we merge it without rebasing on top of develop the commit hash should be the same I think.